modules/hp_timeline/templates/paragraph--hp-timeline.html.twig

```diff
@@ -0,0 +1,23 @@
+<section class="section--padded-top section--padded-bottom"  aria-label="Timeline">
+    {% if hp_show_wrapper %}
+        <section class="compartment">
+    {% endif %}
+            <div>
+                {% if content.field_timeline_title is not null %}
+                    <p class="lead-in">{{ content.field_timeline_title[0] }}</p>
+                {% endif %}
+                {% if content.field_timeline_text is not null %}
+                    <p>{{ content.field_timeline_text[0] }}</p>
+                {% endif %}
+            </div>
+            <div class="timeline">
+                {% for i in 0..content.field_hp_timeline_field|length %}
```

Comment:
This loop seems odd. I get what you are doing, but using 0..content.whatever seems off. Is there a cleaner way to get each item in this array?